packages/camera/camera_avfoundation/ios/Classes/CameraProperties.h

```diff
@@ -30,49 +32,24 @@ extern AVCaptureFlashMode FLTGetAVCaptureFlashModeForFLTFlashMode(FLTFlashMode m
 
 #pragma mark - exposure mode
 
-/// Represents camera's exposure mode. Mirrors ExposureMode in camera.dart.
-typedef NS_ENUM(NSInteger, FLTExposureMode) {
```

Comment:
These enums are no longer needed because we can just use the Pigeon-generated enums instead.